cpptraj and pytraj are not compatible with python3.13 and cython 3.x
see: https://github.com/Amber-MD/pytraj/issues/1677 + https://github.com/Amber-MD/pytraj/issues/1648
--- AmberTools/test/Makefile.orig	2025-08-07 13:56:48.705382000 +0200
+++ AmberTools/test/Makefile	2025-08-07 14:00:13.526533000 +0200
@@ -39,7 +39,7 @@
 ######    Intermediate-level targets:    ############################
 
 fast.serial: clean is_amberhome_defined \
-	fast.cpptraj fast.ambpdb fast.pytraj fast.antechamber fast.mdgx \
+	fast.ambpdb fast.pytraj fast.antechamber fast.mdgx \
 	fast.leap fast.unitcell fast.resp fast.py_resp fast.reduce fast.pbsa fast.gbnsr6 \
 	fast.parmed fast.elsize fast.sqm fast.rism1d \
 	fast.rism3d fast.paramfit \
@@ -86,7 +86,7 @@
 fast.unitcell:
 
 test.serial: clean is_amberhome_defined \
-	test.cpptraj test.pytraj test.antechamber test.mdgx \
+	test.antechamber test.mdgx \
 	test.leap test.unitcell test.resp test.py_resp test.reduce test.pbsa test.gbnsr6 \
 	test.parmed test.elsize test.sqm test.rism1d test.kmmd \
 	test.rism3d test.paramfit test.mm_pbsa \
@@ -96,7 +96,7 @@
 	test.Quick test.modxna test.pyperesp
 
 test.serial.nopython: clean is_amberhome_defined \
-	test.cpptraj test.antechamber test.mdgx \
+	test.antechamber test.mdgx \
 	test.leap test.unitcell test.resp test.reduce test.pbsa test.gbnsr6 \
 	test.elsize test.sqm test.rism1d test.kmmd \
 	test.rism3d test.paramfit test.mm_pbsa \
@@ -106,7 +106,7 @@
 	cd ../src/cpptraj/test && $(MAKE) -k test
 
 test.parallel2: clean is_amberhome_defined \
-	test.cpptraj test.rism3d \
+	test.rism3d \
 	test.Quick.MPI test.mmpbsa.parallel
 
 test.cuda.serial2: clean is_amberhome_defined \
@@ -135,15 +135,15 @@
 
 test.ambermini: test.leap test.sqm test.antechamber test.reduce test.paramfit
 
-test.ptraj:
-	-cd ptraj_rmsa && ./Run.rms
-	-cd ptraj_rms && ./Run.rms
-#	-cd ptraj_les && ./Run.les
-	-cd ptraj_matrix && ./Run.matrix
-	-cd ptraj_order && ./Run.order
-	-cd ptraj_trajectory && ./Run.ptraj
-	-cd ptraj_remd_netcdf && ./Run.ptraj $(NETCDF)
-	-cd ptraj_comprehensive && ./Run.comprehensive $(NETCDF)
+# test.ptraj:
+# 	-cd ptraj_rmsa && ./Run.rms
+# 	-cd ptraj_rms && ./Run.rms
+# #	-cd ptraj_les && ./Run.les
+# 	-cd ptraj_matrix && ./Run.matrix
+# 	-cd ptraj_order && ./Run.order
+# 	-cd ptraj_trajectory && ./Run.ptraj
+# 	-cd ptraj_remd_netcdf && ./Run.ptraj $(NETCDF)
+# 	-cd ptraj_comprehensive && ./Run.comprehensive $(NETCDF)
 
 test.antechamber:
 	cd antechamber && $(MAKE) -k test
@@ -163,8 +163,8 @@
 test.cpptraj.hip:
 	cd ../src/cpptraj/test && $(MAKE) -k test.hip
 
-test.pytraj:
-	cd pytraj && $(MAKE)
+# test.pytraj:
+# 	cd pytraj && $(MAKE)
 
 test.pymsmt:
 	cd pymsmt/mcpb/g03-ff14SB && ./Run.pymsmt
@@ -494,7 +494,7 @@
 test.pysander:
 	cd ../../test/sanderapi && make Python
 
-test.python: test.pysander test.parmed test.pytraj test.pymsmt \
+test.python: test.pysander test.parmed test.pymsmt \
 	     test.cpinutil test.ceinutil test.cpeinutil test.genremdinputs test.fitpkaeo \
 	     test.fixremdcouts test.mmpbsa test.py_resp test.bar_pbsa
 
